controllers/etcd_controller.go

```diff
@@ -959,21 +862,15 @@ func (r *EtcdReconciler) reconcileRoleBinding(ctx context.Context, logger logr.L
 }
 
 func (r *EtcdReconciler) reconcileEtcd(ctx context.Context, logger logr.Logger, etcd *druidv1alpha1.Etcd) (*string, *appsv1.StatefulSet, error) {
-	val := componentetcd.Values{
-		Lease:   componentlease.GenerateValues(etcd),
-		Service: componentservice.GenerateValues(etcd),
+	// Check if Spec.Replicas is odd or even.
+	if etcd.Spec.Replicas&1 == 0 {
```

Comment:
This condition will turn out to be true if replicas is 0, which would prevent the cluster from being hibernated
Can you modify this condition to allow Replicas to be 0?

Reply:
Done